### PR TITLE
Add SaveSettings pump event

### PR DESCRIPTION
**Pull request by ps2**

### CODE REVIEW

File: MinimedKit/PumpEvents/SaveSettingsPumpEvent.swift [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+//  SaveSettingsPumpEvent.swift
+//  RileyLink
+//
+//  Created by Pete Schwamb on 8/26/16.
+//  Copyright © 2016 Pete Schwamb. All rights reserved.
+//
+
+import Foundation
+
+public struct SaveSettingsPumpEvent: TimestampedPumpEvent {
```
Comment (loudnate) [inline MinimedKit/PumpEvents/SaveSettingsPumpEvent.swift]: Do we need this struct? Or can we let `PlaceholderPumpEvent` handle this?

**Comment by loudnate** [comment]
LGTM

**Comment by ps2** [comment]
Yes, forgot I had made that display the json I wanted.  Thanks.

File: MinimedKit/PumpEvents/SaveSettingsPumpEvent.swift (hunk 1 above)
Comment (ps2) [reply MinimedKit/PumpEvents/SaveSettingsPumpEvent.swift]: Yes, forgot I had made that display the json I wanted. Thanks.

**Pull request merged**